rs3-chunkpicker-chunkinfo-export.json

```diff
@@ -158280,8 +158381,8 @@
 				"Revenant bane off hand longsword"
 			],
 			"Barbarian bait[+]": [
-				"feathers+",
```

Comment:
Add `Feathers[+]`

Reply:
Can't nest itemsPlus. Have added colored feathers to this list.